### PR TITLE
Build ipi_deprovision image

### PR DESCRIPTION
**Pull request by hongkailiu**
https://jira.coreos.com/browse/DPTP-471

### CODE REVIEW

File: images/ipi-deprovision/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
+LABEL maintainer="hongkliu@redhat.com"
+
+COPY ipi-deprovision/ipi-deprovision.sh  /usr/bin/ipi-deprovision.sh
```
Comment (petr-muller) [inline images/ipi-deprovision/Dockerfile]: This assumes the context directory is `cmd`, I think this should be `cp cmd/ipi-deprovision/ipi-deprovision.sh destination`

File: images/ipi-deprovision/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Install awscli
+RUN chmod +x /usr/bin/ipi-deprovision.sh && \
+      curl -O https://bootstrap.pypa.io/get-pip.py && \
+      python get-pip.py && \
+      rm -fv get-pip.py && \
+      pip install awscli
```
Comment (petr-muller) [inline images/ipi-deprovision/Dockerfile]: I'm wondering whether we should peg this to a specific version to have control over what we are building upon.

File: images/ipi-deprovision/Dockerfile (hunk 1 above)
Comment (hongkailiu) [reply images/ipi-deprovision/Dockerfile]: Done.

File: images/ipi-deprovision/Dockerfile (hunk 2 above)
Comment (hongkailiu) [reply images/ipi-deprovision/Dockerfile]: Done.

**Pull request merged**